delivery.yaml

```diff
@@ -8,22 +8,25 @@ pipeline:
           apt-get install -y \
           openjdk-8-* \
           maven \
-          git
+          git \
+          python3.5 \
+          python3-pip 
           curl -fLOsS https://delivery.cloud.zalando.com/utils/ensure-docker && sh ensure-docker && rm ensure-docker
 
       - desc: "Build maven package"
         cmd: |
           ./mvnw clean package
 
+          # Our Dockerfiles still need to be compatible with Jenkins deployments and local builds:
+          pip3 install --upgrade scm-source
```

Comment:
why do you need python? Can CDP generate scm-source file for us? At least how does it work for currently generated images?

Reply:
Yes, it can.  The problem is our internal Jenkins cannot as it only acts on pushed artifacts which need an scm-source already in place.  Without out it we need to manually edit local dockerfiles to make our local images deployable, even to playground/staging.  COPY in the dockerfile fails on non-existence.  We could make a small helper script that no-ops on non-existence as an alternative - that would save us the extra CDP build time of installing Python/etc.  That would mean CDP-built PR images would be non-deployable in staging though.  We're using a build tool incompatible with both the deployment tools we use in practice is the problem.  These changes will restore compatibility.

CDP will ignore any already existing scm-source files in favor of its own.